ISSUE_TRIAGE.md

```diff
@@ -12,7 +12,7 @@ Triage helps ensure issues resolve quickly by:
 * Streamlining the development process by preventing duplicate discussions.
 * If you don't have the knowledge or time to code, consider helping with triage. The community will thank you for saving them time by spending some of yours.
 
-** Simplified flowchart diagram of the issue triage process:
+** Simplified flowchart diagram of the issue triage process
```

Comment:
This should either be:

Bold: 
`**Simplified flowchart diagram of the issue triage process**`

or

Heading:
`## Simplified flowchart diagram of the issue triage process`